README.md

```diff
@@ -75,39 +75,41 @@ Examples:
 ```
 ### Using a custom config:
 You can use a custom json file named ```ccube_config.json``` to store configs for ccube,i.e:
-```
+```json
 {
-
- "noLoopSwitch":"true",
+ "noLoopSwitch":true,
  "number":"69",
  "filePath":"void"
-
 }
 ```
 you can change the values to your need.
 
-**Note:Don't change any of the keys(noLoopSwitch,etc) or your config won't work** 
+**Note: Don't remove any of the keys (noLoopSwitch or others) or your config won't work** 
 
-### Build from source:
+### Building from source
 **Build requirements:**
-- Nim(advised to use latest version)
+- Nim (use the latest version for the best results)
 
-To build it from scource.simply: 
+To build it from source, simply: 
 - Clone this repository and cd to the folder.
-- Build it using ```nim c ccube.nim```.if you want to gain optimal speed then use ```-d:danger``` switch which disables all the runtime checks,for more [info](https://nim-lang.org/faq.html).
+- Run ```nim c ccube.nim```. If you want to have the best performance then use `-d:danger` switch which disables all the runtime checks, for more info see [the Nim FAQ](https://nim-lang.org/faq.html). 
+
+### Additional performance flags 
+If your Nim version is 1.4.0 or higher, you can compile using `-d:danger --gc:orc` which is even faster.
```

Comment:
I actually tested this and yes, ORC is faster than the current default GC. See https://nim-lang.org/blog/2020/10/15/introduction-to-arc-orc-in-nim.html and https://nim-lang.org/blog/2020/12/08/introducing-orc.html for more info about ARC and ORC

Reply:
thanks ,good to know